src/navigators/LoggedInStack.tsx

```diff
@@ -71,6 +80,11 @@ export function LoggedInStack() {
       return t('root-stack-accepting-invitation', 'Accepting invitation');
     } else if (loadingJoinCircles) {
       return t('root-stack-waiting-for-circle-join', 'Joining Circles');
+    } else if (shouldWaitForOnUserSignIn) {
+      return t(
+        'root-stack-waiting-for-on-user-signin',
+        'Processing user sign-in',
```

Comment:
For v2, I was thinking we can expose a variable via `developerConfig` to set this loading message. We can also update the `onUserSignIn` hook to dynamically set the loading message.